README.md

```diff
@@ -9,3 +9,57 @@ This repository contains code to download and analyze data from the [Mini-Bird B
 ## Notes on the analysis data files
 
 * Prio
```

Comment:
This line can be deleted. Looks like someone (probably me) started to write something but didn't finish it.